main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+// Users will give us bulk correction files following https://tools.ietf.org/html/draft-google-self-published-geofeeds-02
+// It isn't uncommon for the corrections they list to either match what we already have
+// or to be worse. This script can help us work that out. Right now, it only looks
+// at the ISO country code, but checking more fields should be easy enough.
+import (
+	"fmt"
+	"github.com/oschwald/geoip2-golang"
+	"github.com/yunabe/easycsv"
+	"log"
+	"net"
+	"os"
+	"strings"
+)
+
+// Usage Example:
+// go run go/check_csv_corrections/main.go /path/to/corrections.csv /path/to/mmdbfile.mmdb
+func main() {
+	csvFilename := os.Args[1]
+	mmdbFilename := os.Args[2]
+
+	db, err := geoip2.Open(mmdbFilename)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
```

Comment:
Perhaps if this repo gets a lot of code committed to it in the future, you can have `golangci-lint` run on it. I think if we used the linters we use for `mm_website`, you'd see something like "error not checked on db.Close()` call. Since it's deferred there's not a whole lot we can do there but the pattern we've been using has been to any errors from deferred functions. I think this is not so important and I'm not sure it's worthwhile here, just pointing it out.

Reply:
I'm looking at your code with Will2 and he suggests a decent halfway measure between setting up golangci-lint on Travis/the whole autobuild thing, which is to create a golangci-lint config (TOML) file and running the tool manually and fixing any issues manually.

Reply:
Just to test things out we did the following:

Copy `mm_website`'s .golangci.toml to the root of this repo, then run `golangci-lint run --exclude-use-default=false` to see the linter's output.

Reply:
Using the `.golangci.toml` from `geoipupdate` might be better as the `mm_website` one has some stuff that is specific to that repo.